### ISSUE TITLE
Corriger le bug qui interprète les champs de type textarea en checkbox

### ISSUE THREAD

**Issue by ThierryChampot** [Moventes/bruit.io#61]
**Resources:**
Before submitting an issue, please consult our [docs](https://stenciljs.com/).

**Stencil version:** (run `npm list @stencil/core` from a terminal/cmd prompt and paste output below):

```
 -- (empty)
```

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**

![image](https://user-images.githubusercontent.com/25529818/71814668-6adaf080-307d-11ea-8992-b9b11cf8516d.png)

**Expected behavior:**

"Votre commentaire" should display and behave as a textArea

**Steps to reproduce:**
import and use bruit 1.3.9
try and send a report

**Related code:**

```
  form: [
    {
      label: 'Votre commentaire',
      type: 'textarea',
      required: true
    },
    {
      id: 'agreement',
      type: 'checkbox',
      label: 'Je donne mon accord pour l\'ajout d\'informations techniques à mon message',
      value: null
    }
  ]
```

**Other information:**

**Comment by jonrkarr** [comment]
I'm having the same problem

**Comment by lamstutz** [comment]
fixed in 1.3.11
